src/neo/Wallets/SQLite/SQLiteWalletFactory.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2015-2022 The Neo Project.
+// 
+// The neo is free software distributed under the MIT software license, 
+// see the accompanying file LICENSE in the main directory of the
+// project or http://www.opensource.org/licenses/mit-license.php 
+// for more details.
+// 
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.Plugins;
+using System.IO;
+
+namespace Neo.Wallets.SQLite
+{
+    class SQLiteWalletFactory : IWalletFactory
```

Comment:
Are we going to pull the SQLite code out of NEO in a later PR? Refactoring this code seems like an obvious time to fix https://github.com/neo-project/neo/issues/2678

Reply:
Yes. The SQLite wallet will be moved to neo-modules. But we need to publish Neo first. After the SQLite wallet plugin is released, we can remove the code.